.github/workflows/preDeploy.yml

```
@@ -104,6 +104,17 @@ jobs:
           body: Update version to ${{ steps.bumpVersion.outputs.NEW_VERSION }}
           labels: automerge
 
+      - name: Create Pull Request (staging)
+        uses: peter-evans/create-pull-request@09b9ac155b0d5ad7d8d157ed32158c1b73689109
+        with:
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          author: OSBotify <reactnative@expensify.com>
+          base: staging
+          branch: version-bump-${{ github.sha }}
+          title: Update version to ${{ steps.bumpVersion.outputs.NEW_VERSION }}
```

```suggestion
          title: Update version to ${{ steps.bumpVersion.outputs.NEW_VERSION }} on staging
```

Maybe let's just add `on staging` and `on master` to the end of the titles?

---

Okay, done